app/build.gradle

```diff
@@ -126,4 +126,5 @@ android {
 
 dependencies {
 	implementation 'androidx.preference:preference:1.2.1'
+	implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
```

Comment:
Add a setting for aligning the keyboard to left, center or right. Display it only on screens larger than R.dimen.numpad_max_width, if possible.